src/plugins/postgres_create_tables_like_operator.py

```diff
@@ -0,0 +1,113 @@
+import operator
```

Comment:
Wij hebben al een postgres_copy_operator: https://github.com/Amsterdam/dataservices-airflow/blob/master/src/plugins/postgres_table_copy_operator.py

Deze operator lijk me niet echt anders zijn.

Reply:
`PostgresTableCopyOperator` does three things:

1. copy table definition from _source_ to _target_ (just as the new `PostgresCreateTablesLikeOperator` does)
2. copy over table data from _source_ to _target
3. destroy the _source_ table

Apart from doing more (namely points 2. & 3.),  its use case is different.

The table we want updated should never be destroyed. We want it updated incrementally by `pg_comparator`. However the `PostgresTableCopyOperator` seems designed for situations where you want to replace an entire table with a new copy.

Reply:
3 is already optional.
Maybe it would be better to make data copying also optional instead of creating very similar operator that would confuse people?

Reply:
I'm torn about what to do here. Do we want operators that do one thing well, or do we want operators that can do more than one thing depending on the flags passed to them?

Should the consensus be (also looking for input from @jjmurre) to use the existing `PostgresTableCopyOperator` but parameterize it more, than I'd like to refactor that operator based on the style that I have used for the new `PostgresCreateTablesLikeOperator`, meaning:
- allow it to operate on more than one table
- use the `psycopg2.sql` library for SQL statement formatting instead of plain string formatting

Though not much work I'd like to do that in a separate ticket as this PR is already open for quite some time..

Alternatively we could move to more atomic operators. Meaning, splitting up `PostgresTableCopyOperator` in three new ones:
- the newly `PostgresCreateTablesLikeOperator` (for copying the table definition)
- a to new to create `PostgresTabelDataCopyOperator` (for copying the table data)
- using the `PostgresOperato`r inline with a `DROP TABLE` statement to drop tables.

Basically I'm looking for design advise. However in both cases I'd like to pick up the resulting work in a new ticket.

Reply:
Had another look `PostgresTabelDataCopyOperator`. Turns out it does even more; it also creates junction tables if it finds them. 

Anyway I had the intention of dropping `PostgresCreateTablesLikeOperator` in favor of `PostgresTabelDataCopyOperator` by making the data copying part optional, just as the table dropping part already seemed optional, only to find out that the optional table dropping part could never have worked!

This made me realize I need to be able to test any major modifications I make to this existing operator. And that means running the GOB DAG(s). Plus I take it upon me to document the hell out of whatever I do. That together seems a little much for now (same conclusion I reached yesterday)

I did, however, remove the optional dropping table part in `PostgresTabelDataCopyOperator`. That's something I dared doing without testing. See commit 4d140829f65720181bf19a3ad95244c0a1a1c3e4

Reply:
Okay, reworked `PostgresTabelDataCopyOperator` to make things optional, dropped `PostgresCreateTablesLikeOperator`. Changes tested against `bbga` and `gob_bag_brondocumenten`